### DISCUSSION TITLE
blog/posts/230104-ios-style-picker-3/

### DISCUSSION THREAD

**Discussion by giscus** [ricale/blog#126] [Giscus]
# blog/posts/230104-ios-style-picker-3/

(이전 글들을 읽지 않았더라도 본문을 읽는 데에는 지장이 없다.) 이전 글들에서는 TypeScript 와 React 로 iOS 스타일 Date Picker 를 구현했고, 구현한 컴포넌트를 라이브러리로 만들어 NPM 에 배포했다.

이번에는 라이브러리의 데모…

https://ricale.kr/blog/posts/230104-ios-style-picker-3/

**Comment by H0onnn** [comment]
안녕하세요 ! 좋은 글 잘 읽었습니다.

혹시 시간이랑 요일까지 반영해서 사용할 수 있을까요 ?

**Reply by ricale** [reply]
네 년/월/일 추가하듯이 시간도 추가하면 가능할 것 같습니다
